src/reports/orl/report.ts

```diff
@@ -174,20 +222,26 @@ export function generateORLReport(
 
   // Push data at once into the sheet for performance reasons; Set summary at the end of the file for documentation
   if (rows.length) {
-    datasheet
-      .getRange(5, 1, rows.length, 6 + liabilitiesColumns.length)
+    const range = datasheet
+        .getRange(6, 1, rows.length, rows[0].length);
+
+    activeSpreadsheet.setNamedRange(`ORLTableData`, range);
+    range
       .setValues(rows);
 
     datasheet.appendRow(totalRow);
 
     datasheet
-      .getRange(5, 5, rows.length + 1, 2 + liabilitiesColumns.length)
+      .getRange(6, 5, rows.length + 1, rows[0].length - 5)
```

Comment:
what 6 or 5 means? is an amount of header rows or margin cells? let's create constants to make it clear